packages/workbox-core/INTERNAL/models/messages/messages.mjs

```diff
@@ -0,0 +1,19 @@
+export default {
+  'invalid-type': ({paramName, expectedType, value}) => {
```

Comment:
Add  `= {}` to gracefully deal with the case when someone calls this without passing in any parameters.